ipfx/epochs.py

```diff
@@ -8,10 +8,21 @@
 NOISE_EPOCH = 0.0015
 PRESTIM_STABILITY_EPOCH = 0.5
 POSTSTIM_STABILITY_EPOCH = 0.5
-LONG_RESPONSE_DURATION = 5  # this will count long ramps as completed
 
 
-def get_last_vm_epoch(idx1, hz):
+def get_first_stability_epoch(stim_start_idx, hz):
+
+    num_steps = int(PRESTIM_STABILITY_EPOCH * hz)
+    if num_steps > stim_start_idx-1:
+        num_steps = stim_start_idx-1
+    elif num_steps <= 0:
+        return 0, 0
+    assert num_steps > 0, "Number of steps should be a positive integer"
```

Comment:
Now I'm all for defensive programming but under what condition should this assertion trigger? The `elif` already catches the `<= 0` ones or?

A function comment would also be nice.

Reply:
Yes, looks like the assertion would not trigger. I did not write or intend to touch this function. I removed the assert statement, but this function needs to be modified more generally. So I will leave it as a later issue. Thx